test/e2e/testsuite/testsuite.go

```diff
@@ -539,6 +549,54 @@ func FileExistsInCRCHome(fileName string) error {
 	return err
 }
 
+func BundleIsCachedOrNot(presetName string, isOrNot string) error {
+
+	bundle := ""
+	if presetName == "podman" {
+		bundle = constants.GetDefaultBundle(preset.Podman)
+	} else {
+		bundle = constants.GetDefaultBundle(preset.OpenShift)
+	}
+
+	theFile := filepath.Join(CRCHome, "cache", bundle)
+
+	_, err := os.Stat(theFile)
+
+	if os.IsNotExist(err) && isOrNot == "is" {
+		return fmt.Errorf("file %s does not exists, error: %v ", theFile, err)
+	}
+
+	if err == nil && isOrNot == "is not" {
+		return fmt.Errorf("file %s exists when it should not exist", theFile)
+	}
+
+	return nil
+}
+
+func RemoveBundleFromCache(presetName string) error {
+
+	bundle := ""
+	if presetName == "podman" {
+		bundle := constants.GetDefaultBundle(preset.Podman)
+		dirName := bundle[:len(bundle)-10] // remove `.crcbundle` suffix
+		theFolder = filepath.Join(CRCHome, "cache", dirName)
+		theBundle = filepath.Join(CRCHome, "cache", bundle)
+	} else {
+		bundle := constants.GetDefaultBundle(preset.OpenShift)
+		dirName := bundle[:len(bundle)-10] // remove `.crcbundle` suffix
+		theFolder = filepath.Join(CRCHome, "cache", dirName)
+		theBundle = filepath.Join(CRCHome, "cache", bundle)
+	}
+
+	theFile := filepath.Join(CRCHome, "cache", bundle)
+	err := os.RemoveAll(theFile)
```

Comment:
`theFolder`/`theBundle` are unused.

Reply:
Not sure about this one, but I think as it's now after changes there shouldn't be any unused vars.